.env

```diff
@@ -12,7 +12,7 @@ BUYER="0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a"
 SIGNATORY="0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6"
 #0x90F79bf6EB2c4f870365E785982E1f101E93b906
 
-POLYGONSCAN_API_KEY="HYGZ2HVGXGIBK6NFWWGFBIP8BEE4JWV84G"
+POLYGONSCAN_API_KEY="G6MUHJ3TSMX1RI15W4WC366VD54CXZD2XM"
```

Comment:
Is this a paid / somehow authenticated API key? Maybe we want to keep that secret and not in a public repo.

Reply:
Its a free API key that polygon scan assigns to any account created on their platform. We can add it to secrets, but i dont see a point as its a public api.